api/app/reviews/api.py

```diff
@@ -1,580 +1,650 @@
+        parser.add_argument('reviewer_email', type=str, required=True)
+        args = parser.parse_args()                      
+
+        response_ids = args['response_ids']
+        reviewer_email = args['reviewer_email']
+
+        filtered_response_ids = response_repository.filter_ids_to_event(response_ids, event_id)
+
+        if set(filtered_response_ids) != set(response_ids):
+            return FORBIDDEN
+
+        event = event_repository.get_by_id(event_id)
+
+        reviewer_user = user_repository.get_by_email(reviewer_email, g.organisation.id)
+        if reviewer_user is None:
+            return USER_NOT_FOUND
+        
+        if not reviewer_user.is_reviewer(event_id):
+            _add_reviewer_role(reviewer_user.id, event_id)
+
+        response_reviewers = [ResponseReviewer(response_id, reviewer_user.id) for response_id in response_ids]
+        db.session.add_all(response_reviewers)
+        db.session.commit()
+        
+        if len(response_ids) > 0:
+            email_user(
+                'reviews-assigned',
+                template_parameters=dict(
+                    num_reviews=len(response_ids),
+                    baobab_host=misc.get_baobab_host(),
+                    system_name=g.organisation.system_name,
+                    event_key=event.key
+                ),
+                event=event,
+                user=reviewer_user)
+        return {}, 201
+
+    @event_admin_required
+    def delete(self, event_id):
+        parser = reqparse.RequestParser()
+        parser.add_argument('response_id', type=int, required=True)
+        parser.add_argument('reviewer_user_id', type=int, required=True)
+        args = parser.parse_args()                     
+
+        response_id = args['response_id']
+        reviewer_user_id = args['reviewer_user_id']
+
+        review_form = review_repository.get_review_form(event_id)
+        if not review_form:
+            return REVIEW_FORM_NOT_FOUND
+
+        # If the reviewer has already completed the review, action can't be completed
+        review_response = review_repository.get_review_response(review_form.id, response_id, reviewer_user_id)
+        if review_response:
+            return REVIEW_ALREADY_COMPLETED
+
+        review_repository.delete_response_reviewer(response_id, reviewer_user_id)
+
+        return {}, 200
+
+class ReviewResponseDetailListAPI(restful.Resource):
+    @staticmethod
+    def _serialise_identifier(answer, language):
+        question_translation = answer.question.get_translation(language)
+        if question_translation is None:
+            question_translation = answer.question.get_translation('en')
+            LOGGER.warn('Could not find {} translation for question id {}'.format(language, answer.question.id))
+        
+        return {
+            'headline': question_translation.headline,
+            'value': answer.value_display
+        }
+    
+    @staticmethod
+    def _serialise_scores(review_score, language):

```

Comment:
should be `_serialise_score` (not plural) :)

Reply:
Ah yes good catch!. Fixed now.